openapi/components/schemas/GatewayAccountConfig/DataCash.yaml

```
@@ -62,4 +62,13 @@ allOf:
             minimum: 0
             default: 0
       threeDSecureServer:
-        $ref: ../ThreeDSecureServers/DataCash3dsServers/DataCash3dsServers.yaml
+        type: object
+        description: DataCash 3DS servers.
```

question: should this be singular or plural 
```suggestion
        description: DataCash 3DS server.
```